server/hedley/modules/custom/hedley_reports/hedley_reports.module

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file
+ * Code for the Hedley Reports feature.
+ */
+
+define('HEDLEY_REPORTS_CALCULATE_AGGREGATED_DATA', 'hedley_reports_calculate_reports_data');
+
+/**
+ * Implements hook_menu().
+ */
+function hedley_reports_menu() {
+
+  $items['admin/reports/aggregated-reports'] = array(
+    'title' => 'Aggregated Reports',
+    'description' => 'View Aggregated Reports',
+    'page callback' => 'hedley_reports_aggregated_callback_menu',
+    'access callback' => 'hedley_reports_aggregated_reports_report_access',
+  );
+
+//  $items['admin/reports/aggregated-reports/%/%'] = array(
```

Comment:
Any reason to not delete commented out lines in this file?

Reply:
These are used for the Reports page in a next PR...